rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,153 @@
+can be filed on Zendesk too, or directly in the applicable development
+repository if the reporter is sure of which one to use. Once an issue is
+created in Zendesk there is a tiered support network that kicks in to
+resolve the question.
+
+## Primary Support
+
+The triage person, or team, is responsible for receiving the initial
+support requests from end users, and monitors the Zendesk system daily
+for new tickets. The triager takes ownership of each open ticket, sends
+out a canned response noting that we’ve received the user’s message, and
+looks to see if there is enough information in the ticket to inform the
+response. They will go back to the user for any obvious omissions. Next,
+the triage person will look through the knowledge base for answers to
+known problems. Note that the knowledge base is available to end users,
+and includes the documentation and also FAQs accessible from the Help
+page. If the answer resolves the problem then the issue is closed. If
+not, then the triager needs to ask for help from the secondary support
+team, which includes members from each DCP component or service. It may
+be obvious which team should receive the ticket; if so, then it can be
+assigned directly. If it’s not obvious, there can be some internal
+discussion with the secondary support staff about who should receive the
+ticket.
+
+The triage process will also include tagging the question with basic
+tags (data store/ingest/contribution/etc.). The secondary support person
+to whom the triager asks for help is encouraged to add additional tags
+and to make Github tickets for issues like: new/updated docs,
+new/updated FAQs, code examples, bug reports, etc.
+
+The triager(s) will also work with the secondary support team to create
+and maintain canned responses (how to connect/get involved with the HCA;
+how to submit data, etc.)
+
+The Service Level Objective is to respond to the initial problem
+submission no later than 48 business hours after it is filed.
+
+## Secondary Support
+
+The secondary support team is comprised of 1-2 members from each major
+component or service of the DCP. These representatives should have
+enough technical knowledge to answer most questions. Answers generated
+by the secondary support team will be added to the knowledge base. They
+may require additional information from the person who created the
+ticket. They also may need to consult with technical experts who make up
+the tertiary support teams.
+
+## Tertiary Support
+
+The Tertiary team is the entire DCP team. In the event that the
+secondary support team is not able to respond to the question, anyone
+may be called on to answer it. However, the need for Tertiary support
+should be rare, and all answers will be archived and added to the
+knowledge
+base
+
+### **Acceptance Criteria \[optional\]**
+
+*Acceptance criteria are the conditions that a RFC must satisfy to be accepted by users or other stakeholders.*
```

Comment:
Surely this needs some text?

Reply:
I'll discuss with @kozbo

Reply:
i have removed the optional acceptance criteria section